main.go

```diff
@@ -13,63 +13,35 @@ import (
 	"github.com/bitrise-io/go-steputils/cache"
 	"github.com/bitrise-io/go-steputils/command/gems"
 	"github.com/bitrise-io/go-steputils/command/rubycommand"
+	"github.com/bitrise-io/go-steputils/stepconf"
 	"github.com/bitrise-io/go-utils/command"
 	"github.com/bitrise-io/go-utils/fileutil"
 	"github.com/bitrise-io/go-utils/log"
 	"github.com/bitrise-io/go-utils/pathutil"
-	"github.com/pkg/errors"
 )
 
 // ConfigsModel ...
 type ConfigsModel struct {
-	SourceRootPath  string
-	PodfilePath     string
-	Verbose         string
-	IsCacheDisabled string
+	Command         string `env:"command,opt[install,update]"`
+	SourceRootPath  string `env:"source_root_path,required"`
+	PodfilePath     string `env:"podfile_path"`
```

Comment:
If provided, could be checked to be an existing file.

Reply:
https://github.com/bitrise-steplib/steps-cocoapods-install/pull/68/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R38